src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -618,7 +618,8 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
       detailsProductData,
     } = this.state;
 
-    const productDescription = detailsProductData ? detailsProductData.filter(el => el.name === 'DESCRIPTION')[0]['display-value'].split('. ', 1)[0] : productData._definition[0]['display-name'];
+    const foundDetail = detailsProductData ? detailsProductData.filter(el => el.name === 'DESCRIPTION') : null;
```

Comment:
@yulia-dnistrian made some updates here. She can give you input on the rebase if needed.